interactive/reporter.go

```diff
@@ -2,17 +2,31 @@ package interactive
 
 import (
 	"fmt"
+
 	"github.com/viktomas/godu/core"
 )
 
-func ReportTree(folder *core.File) []string {
-	report := make([]string, len(folder.Files))
+type Line struct {
+	Text     []rune
+	IsMarked bool
+}
+
+func ReportTree(folder *core.File, marker *core.FileMarker) []Line {
+	report := make([]Line, len(folder.Files))
 	for index, file := range folder.Files {
 		name := file.Name
 		if file.IsDir {
 			name = name + "/"
 		}
-		report[index] = fmt.Sprintf("%s %s", formatBytes(file.Size), name)
+		var marking string
+		isMarked := marker.IsMarked(file)
+		if marker.IsMarked(file) {
+			marking = "*"
```

Comment:
I prefer to have it at the very front. Otherwise – having lots of long file names and a non-color terminal – you will not see if a file is marked or not. I'll check on how to fix the jumping.